install_config/cluster_metrics.adoc

```diff
@@ -223,7 +236,7 @@ the secrets which can be used by the deployer:
 
 |*_hawkular-metrics.pem_*
 |The *_pem_* file to use for the Hawkular Metrics certificate. This certificate
-must contain the *hawkular-metrics* host name as well as the publicly available
+must contain the *'hawkular-metrics'* host name as well as the publicly available
```

Comment:
We have had some complaints that it is confusing. I am hoping that by adding the quotes we make it more clear that it has to be the exact string value "hawkular-metrics" and that it does not correspond to some other variable or parameter.